examples/browser/stealth.py

```diff
@@ -28,10 +29,12 @@ async def main():
 	# Default Playwright Chromium Browser
 	normal_browser_session = BrowserSession(
 		# executable_path=<defaults to playwright builtin browser stored in ms-cache directory>,
-		user_data_dir=None,
-		headless=False,
-		# deterministic_rendering=False,
-		# disable_security=False,
+		browser_profile=BrowserProfile(
```

Comment:
why put these in a profile? they're not getting re-used

a profile is just a template for a session, it's only useful if you're re-using the same config over and over